lighthouse-cli/test/smokehouse/test-definitions/forms/form-expectations.js

```diff
@@ -0,0 +1,291 @@
+              autocomplete: '',
+              nodeLabel: 'Select a state\nCA\nMA\nNY\nMD\nOR\nOH\nIL\nDC',
+              snippet: '<select id="state_shipping">',
+            },
+            {
+              id: 'zip_shipping',
+              name: '',
+              placeholder: '',
+              autocomplete: '',
+              nodeLabel: 'input',
+              snippet: '<input type="text" id="zip_shipping">',
+            },
+            {
+              id: 'name_billing',
+              name: '',
+              placeholder: '',
+              autocomplete: '',
+              nodeLabel: 'input',
+              snippet: '<input type="text" id="name_billing">',
+            },
+            {
+              id: 'address_billing',
+              name: '',
+              placeholder: '',
+              autocomplete: 'billing street-address',
+              nodeLabel: 'input',
+              snippet: '<input type="text" id="address_billing" autocomplete="billing street-address">',
+            },
+            {
+              id: 'city_billing',
+              name: '',
+              placeholder: '',
+              autocomplete: '',
+              nodeLabel: 'input',
+              snippet: '<input type="text" id="city_billing">',
+            },
+            {
+              id: 'state_billing',
+              name: '',
+              autocomplete: '',
+              nodeLabel: '\n      Select a state\n      CA\n      MA\n      NY\n      MD\n      OR\n      OH\n   …',
+              snippet: '<select id="state_billing">',
+            },
+            {
+              id: 'zip_billing',
+              name: '',
+              placeholder: '',
+              autocomplete: '',
+              nodeLabel: 'input',
+              snippet: '<input type="text" id="zip_billing">',
+            },
+            {
+              id: 'name_cc2',
+              name: '',
+              placeholder: '',
+              autocomplete: 'cc-name',
+              nodeLabel: 'textarea',
+              snippet: '<textarea type="text" id="name_cc2" autocomplete="cc-name">',
+            },
+            {
+              id: '',
+              name: 'CCNo2',
+              placeholder: '',
+              autocomplete: '',
+              nodeLabel: 'input',
+              snippet: '<input type="text" name="CCNo2">',
+            },
+            {
+              id: 'CCExpiresMonth2',
+              name: '',
+              autocomplete: '',
+              nodeLabel: 'MM\n01\n02\n03\n04\n05\n06\n07\n08\n09\n10\n11\n12',
+              snippet: '<select id="CCExpiresMonth2">',
+            },
+            {
+              id: 'CCExpiresYear',
+              name: '',
+              autocomplete: '',
+              nodeLabel: 'YY\n2019\n2020\n2021\n2022\n2023\n2024\n2025\n2026\n2027\n2028\n2029',
+              snippet: '<select id="CCExpiresYear">',
+            },
+            {
+              id: 'cvc2',
+              name: '',
+              placeholder: '',
```

Comment:
seems like pretty much every `name` and `placeholder` are empty here. feels like we should either remove or add at least one excercising them :)